src/orientation-arm-model.js

```diff
@@ -78,8 +80,12 @@ export default class OrientationArmModel {
   }
 
   setLeftHanded(isLeftHanded) {
-    // TODO(smus): Implement me!
     this.isLeftHanded = isLeftHanded;
+    if(isLeftHanded){
```

Comment:
probably want to add spaces: `if (isLeftHanded) {`

Reply:
Agree, please fix throughout.